### PR TITLE
Kick-Off

### PR DESCRIPTION
**Pull request by fede1608**
# Summary

- Kick-Off express.js application

# Trello Card
[https://trello.com/c/RFx3r32b/1-kick-off](https://trello.com/c/RFx3r32b/1-kick-off)

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+lts/*
```
Comment (mpizzagalli) [inline .nvmrc]: let's use `lts/carbon`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "fec-express-js",
+  "version": "0.1.0",
+  "description": "",
+  "engines": {
+    "node": "8.9.4",
+    "npm": "2.14.2"
```
Comment (mpizzagalli) [inline package.json]: check which version you are actually using and change it here, to avoid future problems with heroku

**Review by mpizzagalli — CHANGES_REQUESTED** [review]
Two little comments

**Comment by fede1608** [comment]
Fixed

**Review by mpizzagalli — APPROVED** [review]
Remember to assign me once I have to look at the PR again, so I get a github notification.

**Pull request merged**